app/main.py

```diff
@@ -1,8 +1,25 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
 
+    def __init__(
+            self,
+            name: str,
+            age: int,
+    ) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
-def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+
+def create_person_list(people: list[dict[str, str]]) -> list[Person]:
+    people_list = [
+        Person(person["name"], int(person["age"])) for person in people
```

Comment:
There are no need cast age to integer. This will be integer by default

Reply:
ok, I know but Pycharm highlight this element "expected int got str instead"